src/history.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+const History = ({ style, viewedAccount, currentPage, loadAccount, selectedNum, setSelected, setPage, loading }) => {
+
+  const transactionList = {
```

Comment:
Move these style object literal declarations outside of `render` method to avoid having to allocate everytime `render` is called. You can move these outside of the component itself into the module.